src/Compilers/CSharp/Test/Semantic/Semantics/NullableConversionTests.cs

```diff
@@ -714,30 +714,30 @@ void Test()
 
             var compilation = CreateCompilation(source);
             compilation.VerifyDiagnostics(
-                // (15,13): error CS0030: Cannot convert type 'double' to 'int?'
+                // (15,13): error CS0221: Constant value 'double' cannot be converted to a 'int?' (use 'unchecked' syntax to override)
```

Comment:
>double [](start = 58, length = 6)

This diagnostic is misleading.  'double' is not a constant value.  Whatever diagnostic is used for this should probably mention overflow.

Reply:
I made a change the error message to:
``error CS0221: Constant value '1.7976931348623157E+308' cannot be converted to a 'int?' (use 'unchecked' syntax to override)``